customer-address.go

```diff
@@ -0,0 +1,30 @@
+package httpshopify
+
+import (
+	"fmt"
+
+	"github.com/MOHC-LTD/httpshopify/internal/http"
+)
+
+type customerAddressRepository struct {
+	client    http.Client
+	createURL func(endpoint string) string
+}
+
+func newCustomerAddressRepository(client http.Client, createURL func(endpoint string) string) customerAddressRepository {
+	return customerAddressRepository{
+		client,
+		createURL,
+	}
+}
+
+func (r customerAddressRepository) Delete(id int64, addressID int64) error {
+	url := r.createURL(fmt.Sprintf("customers/%v/addresses/%v.json", id, addressID))
+
+	_, _, err := r.client.Delete(url, nil)
+	if err != nil {
+		return err
```

Comment:
What will you do if the delete fails?

Reply:
I guess returning the error only and then is handled accordingly in FEMS? I took as example the Delete in `fulfillment-events.go`. You think we should add something else?

Reply:
Don't add anything unless you are using it. So if you are happy to have the error as it is and catch it then its fine.

Reply:
I think so.. According to designs, the FE is only providing a generic error message to the user when something goes wrong with deleting an address. It doesn't give any details. The network tab probably would just indicate whatever we send from the FEMS 👍🏻